File: 0217-contains-duplicate/0217-contains-duplicate.java
import java.util.*;
class Solution {
    public boolean containsDuplicate(int[] nums) {
        Arrays.sort(nums);
        for(int i =0 ;i<nums.length;i++){
            if(i==nums.length-1){continue;}
           if(nums[i]==nums[i+1]){
            return true;
           }
        }
        return false;
    }
}

File: 0217-contains-duplicate/0217-contains-duplicate.js
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var containsDuplicate = function (nums) {
    // const seen = new Set();

    // for (let num of nums) {
    //     if (seen.has(num)) {
    //         return true; 
    //     }
    //     seen.add(num);
    // }

    // return false;
    nums.sort((a, b) => a - b)
    for (let i = 0; i < nums.length; i++) {
        if (nums[i] == nums[i + 1]) {
            return true
        }
    }
    return false
};
